PxWeb/appsettings.Release.json

```diff
@@ -39,7 +39,32 @@
     "CacheTime": 86400,
     "SearchEngine": "Lucene",
     "PageSize": 20,
-    "BaseURL": "#{BASEURL}#"
+    "BaseURL": "#{BASEURL}#",
+    "OutputFormats": [
+      "xlsx",
+      "xlsx_doublecolumn",
+      "csv",
+      "csv_tab",
+      "csv_tabhead",
+      "csv_comma",
+      "csv_commahead",
+      "csv_space",
+      "csv_spacehead",
+      "csv_semicolon",
+      "csv_semicolonhead",
+      "csv2",
+      "csv3",
+      "json",
+      "json_stat",
```

Comment:
PR #30 changed the name to json-stat and json-stat2.

Reply:
This is now fixed